packages/patrol_cli/lib/src/features/test/native_test_runner.dart

```diff
@@ -95,8 +135,17 @@ class NativeTestRunner extends TestRunner implements Disposable {
             continue;
           }
 
+          // Build once for every target, no matter how many repeats.
+          try {
+            await builder(target, device);
+          } catch (_) {
+            targetRuns.add(TargetRunStatus.failedToBuild);
+            continue;
+          }
+
           for (var i = 0; i < _repeats; i++) {
             if (_disposed) {
+              print('disposed!');
```

Comment:
do you just print like that?

Reply:
Nope, I use a slightly customized instance of [mason_logger.Logger](https://pub.dev/documentation/mason_logger/latest/mason_logger/Logger-class.html).

This is a mistake, thx.